code/__DEFINES/nukes.dm

```diff
@@ -0,0 +1,6 @@
+#define NUKE_STAGE_NONE 0
```

Comment:
This would be a fine example of putting it into the file itself rather than another file

Reply:
Undef stuff that you put into files at the end of it.
If you use the define in more places outside of the file, then leave it in `__DEFINES`.